test/specs/utils/util_test.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
```

Comment:
?

Reply:
I admit I don't know the policy here, having just copied the license header from another file. Should I omit @yuya-oc's copyright on this file?

Reply:
Well he didn't write it. No idea why it's on the others. I think legally it's fine because the CLA doesn't sign over copyright just an indefinite license.

Reply:
Ah, now I know why:
```
/root/mattermost-desktop/test/specs/utils/util_test.js
  3:1  error  incorrect header  header/header
```

Fixing this by making the `header/header` rule apply to a hard-coded list of files, and going forward will only require the mattermost license on new files.